tools/gen_locale.dart

```diff
@@ -14,6 +11,8 @@
 // this script (the first set for _canonicalizeLanguageCode and the second set
 // for _canonicalizeRegionCode).
 
+// ignore_for_file: avoid_print
```

Comment:
This shouldn't be necessary since there's a global opt-out for the `tools` directory in https://github.com/flutter/engine/blob/main/tools/analysis_options.yaml.

Reply:
My IDE was complaining about this but maybe it wasn't properly configured?